sihl_core/lib/run.ml

```diff
@@ -7,21 +7,24 @@ module type APP = sig
 
   val namespace : string
 
-  val config : Config.Schema.t
+  val config : unit -> Config.Schema.t
```

Comment:
Should this be async?

Reply:
Actually should all of the functions here be async?

Reply:
Good question. I just did the simplest thing, but we can make it async before 1.0.0.